### PR TITLE
New version: Microsoft.PowerShell.Preview version 7.5.0.101

### PR DESCRIPTION
**Pull request by anamnavi**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/191873)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

File: manifests/m/Microsoft/PowerShell/Preview/7.5.0.101/Microsoft.PowerShell.Preview.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- pwsh
+ReleaseDate: 2024-11-14
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.5.0-rc.1/PowerShell-7.5.0-rc.1-win-x64.msi
+  InstallerSha256: EF92C1B1759F1CC33955A6CDC20AC329D88030FF6862C37991E089E86717CC69
+  ProductCode: '{76B09914-8F51-46D6-B700-C0E6C9CF6459}'
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/Preview/7.5.0.101/Microsoft.PowerShell.Preview.installer.yaml:21]:
```suggestion
  ProductCode: '{76B09914-8F51-46D6-B700-C0E6C9CF6459}'
  AppsAndFeaturesEntries:
  - Publisher: Microsoft Corporation
    DisplayName: PowerShell 7-preview-x64
```

File: manifests/m/Microsoft/PowerShell/Preview/7.5.0.101/Microsoft.PowerShell.Preview.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+- Architecture: x86
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.5.0-rc.1/PowerShell-7.5.0-rc.1-win-x86.msi
+  InstallerSha256: 63E014188725BA7E0608228393C0E2B7F0AF9D00650FD2050ED8B07AAEBA11A9
+  ProductCode: '{EAE8518D-EBB8-4162-901D-A29045FF293A}'
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/Preview/7.5.0.101/Microsoft.PowerShell.Preview.installer.yaml:25]:
```suggestion
  ProductCode: '{EAE8518D-EBB8-4162-901D-A29045FF293A}'
  AppsAndFeaturesEntries:
  - Publisher: Microsoft Corporation
    DisplayName: PowerShell 7-preview-x86
```

File: manifests/m/Microsoft/PowerShell/Preview/7.5.0.101/Microsoft.PowerShell.Preview.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- Architecture: arm64
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.5.0-rc.1/PowerShell-7.5.0-rc.1-win-arm64.msi
+  InstallerSha256: 1D290DAE570D51D752D5497CB8BBC0ECDCDDBE1CB81893F3AE690E348B0430D9
+  ProductCode: '{71510522-4933-4000-8454-C23921767EF5}'
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/Preview/7.5.0.101/Microsoft.PowerShell.Preview.installer.yaml:29]:
```suggestion
  ProductCode: '{71510522-4933-4000-8454-C23921767EF5}'
  AppsAndFeaturesEntries:
  - Publisher: Microsoft Corporation
    DisplayName: PowerShell 7-preview-arm64
```

File: manifests/m/Microsoft/PowerShell/Preview/7.5.0.101/Microsoft.PowerShell.Preview.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- Architecture: arm
+  InstallerUrl: https://github.com/PowerShell/PowerShell/releases/download/v7.5.0-rc.1/PowerShell-7.5.0-rc.1-win-arm64.msi
+  InstallerSha256: 1D290DAE570D51D752D5497CB8BBC0ECDCDDBE1CB81893F3AE690E348B0430D9
+  ProductCode: '{71510522-4933-4000-8454-C23921767EF5}'
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/Preview/7.5.0.101/Microsoft.PowerShell.Preview.installer.yaml:33]:
```suggestion
  ProductCode: '{71510522-4933-4000-8454-C23921767EF5}'
  AppsAndFeaturesEntries:
  - Publisher: Microsoft Corporation
    DisplayName: PowerShell 7-preview-arm
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
`AppsAndFeaturesEntries` from previous manifest were not carried forward here. They should be added to assist WinGet in properly matching the package

**Comment by anamnavi** [comment]
@mdanish-kh oops yes thanks for catching that!!

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**